blockchain/blocks/src/tipset.rs

```diff
@@ -47,6 +47,15 @@ impl PartialEq for Tipset {
     }
 }
 
+impl quickcheck::Arbitrary for TipsetKeys {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        let header = BlockHeader::arbitrary(g);
+        Self {
+            cids: vec![*header.cid()],
```

Comment:
You don't have to create a block header just to get a CID. Instead, just generate a vector of random CIDs.

Reply:
👍